src/buildblock/ProjDataInfo.cxx

```diff
@@ -177,9 +177,41 @@ ProjDataInfo::set_tof_mash_factor(const int new_num)
     {
       tof_mash_factor = new_num;
       if (tof_mash_factor > scanner_ptr->get_max_num_timing_poss())
+        {
         error("ProjDataInfo::set_tof_mash_factor: TOF mashing factor (" + std::to_string(tof_mash_factor)
               + +") must be smaller than or equal to the scanner's number of max timing bins ("
               + std::to_string(scanner_ptr->get_max_num_timing_poss()) + ").");
+        }
+      else if (tof_mash_factor == num_tof_bins)
```

Comment:
this turns out to fail in test_time_of_flight. The reason is that this is called in the constructor, where `num_tof_bins` is still 1. We should check  on the scanner num_tof_bins instead. However, I don't think we need this at all, as the normal code will do the same.

Reply:
Let's try that then

Reply:
fine for now. Can always revisit later.